### ISSUE TITLE
Group assigned

### ISSUE THREAD

**Issue by changux** [giddie/redmine_default_assign#20]
Hi.
Some way to default assign a group?

Regards.

**Comment by giddie** [comment]
Yes, this is working fine for me. Can you describe how this is not working for you?

**Comment by shawndibble** [comment]
I can select groups in terms of global assignees (administration -> configuration settings for plugin).

However, I am unable to select groups on individual projects (even if the group is a member of the project).

**Comment by giddie** [comment]
Do you have group assignment enabled for the project? A project won't necessarily permit group assignment---you have to enable it in the project settings.

**Comment by shawndibble** [comment]
Yes, I did. The thing I was doing was I added the group under the members tab, then immediately went to Information to add the Default Assignee instead of refreshing the page first. After I did an actual refresh of the page, the list updated.

**Comment by giddie** [comment]
OK, that's helpful, thanks. I'd welcome patches to improve that behaviour, but as it's a one-time configuration and the workaround is quite discoverable, I'm unlikely to find time to fix this myself.
